### PR TITLE
test to check on missing maxScore value from Solr #448 [JIRA: RIAK-1498]

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
Addes test to show https://github.com/basho/yokozuna/pull/447#issue-55647850 is not occurring w/ erlang pb client. Also updates kvc lib in case we need defaults. So, all in all, a minor update.

### CODE REVIEW

**Comment by cmeiklejohn** [comment]
I will take this review.

**Comment by zeeshanlakhani** [comment]
@cmeiklejohn, cool. I've set this back to develop on riak_kv, so things should work as they should when the other develop branches are updated.

**Comment by cmeiklejohn** [comment]
Is there a test for this?

**Comment by zeeshanlakhani** [comment]
This was a real simple edit based on the right types, but I'll add an eunit... and also wait for the update to the develop branch.

**Comment by cmeiklejohn** [comment]
The test will ensure no one removes it in the future.

**Comment by zeeshanlakhani** [comment]
@cmeiklejohn agreed. Will do.

**Comment by zeeshanlakhani** [comment]
Test proves that the erlang client (pb) works w/o the defaults. Adding the test and updating kvc for things like defaults if we need it. @angrycub will keep the current yz_patch and take a further look on what's going on w/ the difference between the java client (where max_score can be an empty list) and erlang client.

File: riak_test/yz_pb.erl [hunk 1]
```diff
@@ -244,3 +247,28 @@ confirm_stored_fields(Cluster) ->
...
+    Body = <<"{\"age_i\":5}">>,
+    Params = [{sort, <<"age_i asc">>}],
+    HP = select_random(host_entries(rt:connection_info(Cluster))),
+    URL = bucket_url(HP, Bucket, "multivalued"),
+    lager:info("Storing to bucket ~s", [URL]),
+    {Host, Port} = HP,
+    %% populate a value
+    {ok, "204", _, _} = ibrowse:send_req(URL, [?CT_JSON], put, Body),
+    %% Sleep for soft commit
+    timer:sleep(1100),
```
Comment (cmeiklejohn) [inline riak_test/yz_pb.erl]: Anything better you can do here?  Seems like it could trigger a race.

**Comment by cmeiklejohn** [comment]
Something seems weird here; I'm only seeing the addition of the test and not the code?

File: riak_test/yz_pb.erl [hunk 2]
```diff
@@ -244,3 +247,28 @@ confirm_stored_fields(Cluster) ->
...
+    Search = <<"age_i:5">>,
+    {ok, Pid} = riakc_pb_socket:start_link(Host, (Port-1)),
+    {ok,{search_results,[{Index,Fields}],Score,_Found}} =
+        riakc_pb_socket:search(Pid, Index, Search, Params),
+    ?assertNotEqual(Score, []),
+    ?assertNotEqual(Score, 0),
```
Comment (cmeiklejohn) [inline riak_test/yz_pb.erl]: Why not assert what the score should be?  Do we know?

**Comment by Basho-JIRA** [comment]
Updated Note: https://github.com/basho/yokozuna/pull/448#issuecomment-73633332 

**_[posted via JIRA by Zeeshan Lakhani]**_

File: riak_test/yz_pb.erl (hunk 1 above)
Comment (zeeshanlakhani) [inline riak_test/yz_pb.erl]: I was just keeping to the rest of the set of tests, currently. The initial patch was not due to a race cond.

File: riak_test/yz_pb.erl (hunk 2 above)
Comment (zeeshanlakhani) [inline riak_test/yz_pb.erl]: We can't determine the exact score from solr, but I'll push a line of cleanup right now actually :).

**Comment by zeeshanlakhani** [comment]
I've rebased things to focus the commits on purely adding the test in response to the issue brought up in #447.

**Comment by zeeshanlakhani** [comment]
@cmeiklejohn sanity of commits now preserved, timeout removed.

**Comment by zeeshanlakhani** [comment]
Just needs 2.0 -> develop riak_kv.

**Comment by zeeshanlakhani** [comment]
@borshop: retry 431ff5ec97430a09220924a689993876869b6bb7

**Comment by cmeiklejohn** [comment]
That branch needs to be rebased and squashed before we consider merging.

**Comment by zeeshanlakhani** [comment]
@cmeiklejohn rebased. just need the fix in the mecks for green.

**Comment by cmeiklejohn** [comment]
@borshop: retry

**Comment by cmeiklejohn** [comment]
Bors is confused.  Open a new PR with the newly rebased commit?

**Comment by zeeshanlakhani** [comment]
@cmeiklejohn bors won't retry on pr comments. It will only retry on a commit that has been initially +1'ed. But, I'll open a new PR to make things easier.

**Pull request closed without merging**